trawl.el

```diff
@@ -364,11 +364,12 @@
     (when (> trawl--error-count errors-before)
       (trawl--show-errors))))
 
-(defun trawl--tree (dir)
+(defun trawl--tree (dir &optional excludes)
   (let ((excludes (or excludes "$^")))
```

Comment:
Note that "$^" is a regexp that matches that literal string (dollar followed by caret). Unfortunately, trawl isn't smart to find that error, since it would involve some data-flow analysis, but xr-lint warns about it if run directly.

If you want a regexp that never matches anything, try something like "a\\`".

Reply:
Sorry, that should have been ```"a\\`"``` —the markdown parser ate some characters.

Reply:
Yes I wanted something that never matches anything. I'll correct the patch.